src/core_plugins/elasticsearch/index.js

```diff
@@ -42,7 +42,8 @@ export default function (kibana) {
         certificateAuthorities: array().single().items(string()),
         certificate: string(),
         key: string(),
-        keyPassphrase: string()
+        keyPassphrase: string(),
+        alwaysPresentCertificate: boolean().default(false),
```

Comment:
I think we'll need this in monitoring too, iirc it uses its own copy if url is set.

Reply:
Cool call, thanks JBudz